backend/typescript/services/implementations/emailService.ts

```diff
@@ -0,0 +1,28 @@
+import nodemailer from "nodemailer";
+import NODEMAILER_CONFIG from "./nodemailer.config";
+import IEmailService from "../interfaces/emailService";
+import Logger from "../../utilities/logger";
+
+const EMAIL_FROM = `"Display Name" <address@email.org>`;
+const transporter = nodemailer.createTransport(NODEMAILER_CONFIG);
+
+class EmailService implements IEmailService {
+  /* eslint-disable class-methods-use-this */
+  async sendEmail(to: string, subject: string, html: string): Promise<void> {
```

Comment:
> Should the interface for sendEmail include a "from" parameter? Or is it reasonable to assume that they will all be sent from a single email

Yeah I think it would be nice to allow "from" to be configurable, seems like a reasonable scenario that a project would require sending from multiple email accounts. I suggest adding a constructor to instantiate the transporter, this way users can change the account by using a new instance of `EmailService`

```ts
constructor(nodemailerConfig: NodeMailerConfig) {
    this.transporter = nodemailer.createTransport(nodemailerConfigs);
}
```